apps/tlv-app/grails-app/assets/stylesheets/index.css

```diff
@@ -47,3 +47,18 @@
     transition: all 1s;
     -webkit-transition: all 1s;
 }
+
+.unclassified {
+    background-color: green !important;
+    color: white !important;
+}
+
+.secret {
+    background-color: red !important;
+    color: white !important;
+}
+
+.top-secret {
+    background-color: yellow !important;
+    color: black !important;
+}
```

Comment:
Needs a default class, possibly the highest classification in the system.

Reply:
Can we move this to omar-security/plugins/omar-security-plugin/grails-app/assets/stylesheet/security.css?